### PR TITLE
mainscreen

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (114)

### CODE REVIEW

File: quiz-app/src/components/StartScreen.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+
+function StartScreen({start}){
+     return (
+          <div className="Start-Screen">
+               {/* <h2>Start Quiz</h2>
```
Comment (tanishq20) [inline quiz-app/src/components/StartScreen.js]: Remove cmts if not neended.

**Review by tanishq20 — APPROVED** [review]
Functionality is well written. Learned something new.

File: quiz-app/src/components/StartScreen.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+               <p className="title">INDIA MOST POPULAR COMEDY TELEVSION SHOW.</p> */}
```
Comment (Bharatrathod-81) [inline quiz-app/src/components/StartScreen.js]: comment can be removed

**Review by Bharatrathod-81 — APPROVED** [review]

**Review by therajatg — APPROVED** [review]
looks good

File: quiz-app/src/App.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
-
+import {useState} from "react";
 import Navbar from "./components/Navbar.js";
```
Comment (Priya-Patel31) [inline quiz-app/src/App.js:2]: You can have a file import all these components and import that file here.

File: quiz-app/src/index.css [hunk 4]
```diff
@@ -110,6 +110,16 @@ code {
   text-align: center;
 }
 
+.Quiz-container{
+  position: relative;
+  margin: 50px auto 20px;
+  width: 95%;
+  max-width: 650px;
```
Comment (Priya-Patel31) [inline quiz-app/src/index.css:120]: change px to rems

File: quiz-app/src/index.css [hunk 5]
```diff
@@ -110,6 +110,16 @@ code {
...
+  min-height: 350px;
+  background: #fff;
```
Comment (Priya-Patel31) [inline quiz-app/src/index.css]: use variables for these colors.

File: quiz-app/src/index.css [hunk 6]
```diff
@@ -140,16 +150,101 @@ code {
...
+}
+
+.quiz-screen .question .main{
+  padding: 20px 40px;
+}
+
+.quiz-screen .question .main .title span{
+  font-size: 15px;
+  color: #555;
+  font-weight: 600px;
```
Comment (Priya-Patel31) [inline quiz-app/src/index.css]: font weight doesn't need to be in px it should be just 600.

File: quiz-app/src/index.css [hunk 7]
```diff
@@ -140,16 +150,101 @@ code {
...
+  margin: 40px 0px 20px;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  gap: 20px;
+}
+
+.quiz-screen .question .main .options .option{
+  width:  calc(50% - 20px);
+  border: 1px solid #bbb;
```
Comment (Priya-Patel31) [inline quiz-app/src/index.css:205]: use variables for colors.

File: quiz-app/src/index.css [hunk 8]
```diff
@@ -140,16 +150,101 @@ code {
...
+  padding: 10px;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.quiz-screen .question .main .options .option.active{
+  background: #1428cd;
```
Comment (Priya-Patel31) [inline quiz-app/src/index.css]: use variables for colors .

File: quiz-app/src/index.css [hunk 9]
```diff
@@ -140,16 +150,101 @@ code {
...
+  color: #fff;
+}
+.quiz-screen .question .control{
+  padding: 10px 20px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  border-top: 1px solid #eee;
```
Comment (Priya-Patel31) [inline quiz-app/src/index.css]: same as above

File: quiz-app/src/index.css [hunk 10]
```diff
@@ -140,16 +150,101 @@ code {
...
+}
+
+.result-screen{
+  width: 100%;
+  height: 350px;
```
Comment (Priya-Patel31) [inline quiz-app/src/index.css:225]: change in to rems

File: quiz-app/src/components/Mainscreen.js [hunk 11]
```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+
+import QuestionList from "../Data/Data.json";
+
```
Comment (Priya-Patel31) [inline quiz-app/src/components/Mainscreen.js]: remove spaces

**Review by Priya-Patel31 — APPROVED** [review]
Everything seems good to me . Just have suggested few changes you can go through them .
Change all the colors with color variables and px to rems .

File: quiz-app/src/components/Mainscreen.js (hunk 11 above)
Comment (iamsaurav28) [reply quiz-app/src/components/Mainscreen.js]: ok done

File: quiz-app/src/index.css (hunk 10 above)
Comment (iamsaurav28) [reply quiz-app/src/index.css:225]: ok

File: quiz-app/src/index.css (hunk 5 above)
Comment (iamsaurav28) [reply quiz-app/src/index.css]: ok

File: quiz-app/src/index.css (hunk 9 above)
Comment (iamsaurav28) [reply quiz-app/src/index.css]: okk

**Pull request merged**